### PR TITLE
Use AVA for test

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by TrySound** [comment]
Damn, wtf. Tests are passed, but powershell is failed.

**Comment by TrySound** [comment]
@MoOx  Done!)

File: appveyor.yml [hunk 1]
```diff
@@ -16,5 +16,4 @@ install:
 test_script:
   - node --version
   - npm --version
-  - ps: "npm test # PowerShell"
```
Comment (MoOx) [inline appveyor.yml]: oh nice, I recently did the same for some repo :+1:

**Comment by MoOx** [comment]
Thanks!

**Pull request merged**